backend/src/main/java/com/bakdata/conquery/io/storage/xodus/stores/XodusStore.java

```diff
@@ -24,16 +24,19 @@
 	private final Collection<Store>  openStores;
 	private final Consumer<Environment> envCloseHook;
 	private final Consumer<Environment> envRemoveHook;
+	@Getter
+	private String name;
 
-	public XodusStore(Environment env, IStoreInfo storeId, Collection<Store> openStoresInEnv, Consumer<Environment> envCloseHook, Consumer<Environment> envRemoveHook) {
+	public XodusStore(Environment env, String name, Collection<Store> openStoresInEnv, Consumer<Environment> envCloseHook, Consumer<Environment> envRemoveHook) {
```

Comment:
Eine Sache die ich noch angehen wollte war, dass das eine komische inversion of control ist, dass die stores die collection pflegen

Reply:
Ja da hast du recht, das habe ich komisch gemacht

Reply:
Ich habe es weg gemacht